internal/pkg/template/workload.go

```diff
@@ -522,11 +511,24 @@ func (cfg *ALBListener) RulePaths() []string {
 	return rulePaths
 }
 
-// ServiceConnect holds configuration for ECS Service Connect.
-type ServiceConnect struct {
+// ServiceConnectServer defines the container name and port which a service routes Service Connect through.
+type ServiceConnectServer struct {
+	Name  string
+	Port  string
 	Alias *string
 }
 
+// ServiceConnectOpts holds configuration for ECS Service Connect.
+type ServiceConnectOpts struct {
+	Server  *ServiceConnectServer
```

Comment:
Because `ServiceConnect` can be enabled, but `ServiceConnectOpts` manages both a client/server, so the server option will need to be nil for the client.